src/components/Dashboard/ProjectHeader.tsx

```diff
@@ -131,9 +131,11 @@ export default function ProjectHeader() {
                     paddingRight: 10,
                   }}
                 >
-                  ID: {projectId.toNumber()}{' '}
+                  <Trans>ID: {projectId.toNumber()}</Trans>{' '}
```

Comment:
should the projectId be inside the <Trans/> component?

Reply:
Yeah it should. In some languages they structure 'something: something else' differently